tests/formats/kzg/compute_kzg_proof.md

```diff
@@ -10,14 +10,15 @@ The test data is declared in a `data.yaml` file:
 input:
   blob: Blob -- the data blob representing a polynomial
   z: Bytes32 -- bytes encoding the BLS field element at which the polynomial should be evaluated
-output: KZGProof -- The KZG proof
+output: Tuple[KZGProof, Bytes32] -- The KZG proof and the value y = f(z)
```

Comment:
The test vector results are more like the Python array:
```yaml
output: ['0xc00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000',
  '0x00000000fffffffffe5bfeff02a4bd5305d8a10908d83933487d9d2953a7ed73']
```

We could [hack yaml constructor](https://stackoverflow.com/a/48452275) to make it looks like Tuple, but it may add more overhead for clients to load the files.

It's more correct to just describe it as [YAML sequence](https://www.educative.io/courses/introduction-to-yaml/N881Nq9pVY8#Flow-style)

```suggestion
output: [KZGProof, Bytes32] -- YAML sequence. The first element is KZG proof, and the second element is y = f(z)
```

or just remove "Tuple"

```suggestion
output: [KZGProof, Bytes32] -- The KZG proof and the value y = f(z)
```

Reply:
I actually don't see this as a problem -- a "tuple" is a sequence type, just one that cannot be appended to or deleted from. YAML doesn't distinguish a tuple from a list because it doesn't have any notion of appending or deleting elements.

So my preference is leaving as is, because it is conceptually a tuple -- it's always exactly two elements.

Reply:
I prefer explicitly calling it a tuple too.